ocrd_tesserocr/common.py

```diff
@@ -83,16 +195,25 @@ def image_from_page(workspace, page,
         page_image = workspace.resolve_image_as_pil(
             alternative_image[-1].get_filename())
     elif border:
-        page_image = page_image.crop(
+        # create a mask from the page polygon:
+        page_image = image_from_polygon(page_image, page_polygon)
+        # recrop into page rectangle:
+        page_image = crop_image(page_image,
             box=(page_xywh['x'],
                  page_xywh['y'],
                  page_xywh['x'] + page_xywh['w'],
                  page_xywh['y'] + page_xywh['h']))
-        # FIXME: mask away all GraphicRegion, SeparatorRegion etc which
-        # could overlay any text regions
+        # FIXME: uncomment as soon as we get @orientation in PageType:
```

Comment:
See above.

Reply:
See [above](https://github.com/OCR-D/ocrd_tesserocr/pull/56#issuecomment-511425213): I could rewrite so page orientation gets active automatically as soon as the new schema version is merged into core.